superset/connectors/druid/views.py

```diff
@@ -61,9 +62,26 @@ class DruidColumnInlineView(CompactCRUDMixin, SupersetModelView):  # noqa
             True),
     }
 
+    def pre_update(self, col):
+        # If a dimension spec JSON is given, ensure that it is
+        # valid JSON and that `outputName` is specified
+        if col.dimension_spec_json:
```

Comment:
While I like the validation here in general, I feel like its a bit arbitrary that we validate that 
`dimension_spec_json` is valid but we don't do any validation for SQL columns or SQL/Druid metrics.

Reply:
I noticed as well. Should we starting adding validation (or 'warnings') for other things as well?